virtual-desktop/src/app/window-manager/mvd-window-manager/launchbar/launchbar/launchbar.component.ts

```diff
@@ -181,12 +181,23 @@ export class LaunchbarComponent {
       this.applicationManager.spawnApplication(this.propertyWindowPluginDef,this.getAppPropertyInformation(plugin));
     }  
   }
+
+  openStandalone(item: LaunchbarItem): void {
+    const plginType:string = item.plugin.getFramework();
+    if (plginType === 'iframe') {
+      // Still allows IFrames to comprehend URL parameters if address is copy/pasted later
+      window.open(`${location.origin}/ZLUX/plugins/${item.plugin.basePlugin.getIdentifier()}/web/`);
+    } else {
+      window.open(`${location.href}?pluginId=${item.plugin.basePlugin.getIdentifier()}`);
+    }
+  }
   
   onRightClick(event: MouseEvent, item: LaunchbarItem): boolean {
     var menuItems: ContextMenuItem[];
     if (item.instanceCount == 1) {
         menuItems = [
           { "text": this.translation.translate("Open New"), "action": ()=> this.openWindow(item)},
+          { "text" : "Open Standalone", "action": () => this.openStandalone(item)},
```

Comment:
Hey this functionally works, which is great, but disrupts internationalization

![image](https://user-images.githubusercontent.com/20528015/62633147-aff9cb80-b901-11e9-8c16-756e34ba9c3d.png)

So let's touch base on Slack when you see this, change it to accept translations (it won't break, if there's no translations present, it will just use the default supplied text string) and I'll touch base with our translators to give you the strings that you can add to the i18n assets

Reply:
Let's get some translations into this PR as your follow-up.